main.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+function makeSortable (elem) {
+  var timeOnPage = localStorage.getItem("timeOnPage");
+  let append = document.createElement('p');
```

Comment:
append is a bad name for element because the same as DOM API method
use `counterContainer` etc.